packages/report-server/src/routes/FetchReportRoute.ts

```diff
@@ -63,6 +80,12 @@ export class FetchReportRoute extends Route<FetchReportRequest> {
     };
 
     const reportBuilder = new ReportBuilder(reqContext).setConfig(report.config);
-    return reportBuilder.build();
+    const reportResult = await reportBuilder.build();
+
+    return {
+      ...reportResult,
+      // add the item type son we can use the appropriate types
+      type: itemType,
```

Comment:
I tossed up where to add this, but because it need to apply to LESMIS as well as Tupaia, I added it here for non-legacy reports and the web config server for legacy reports, so that it got pulled through everywhere

Reply:
Even though I’ve reviewed, I don’t think I can give much input on this. Perhaps best to ping Rohan if you’re still undecided?

Reply:
@rohan-bes would you mind casting your eye over this part?

Reply:
Feels a bit awkward having this in the `report-server`, which should really just be dealing with the data side of the viz. Wanna do a little pairing on this @alexd-bes